1-js/09-classes/07-mixins/article.md

```diff
@@ -136,7 +136,7 @@ let eventMixin = {
   },
 
   /**
-   * Cancel the subscription, usage:
+   * :لغو کردن تعهد، کاربرد
```

Comment:
Needs more attention (translation of "subscription").

Reply:
```suggestion
   * :لغو کردن اشتراک، استفاده
```